cli/flox/src/commands/environment.rs

```diff
@@ -602,6 +603,10 @@ impl Activate {
         if in_place {
             Self::activate_in_place(&shell, &exports, &activation_path);
 
+            let env = default_nix_subprocess_env(true);
+            env.iter()
+                .for_each(|(k, v)| println!("export {}='{}'", k, v));
+
```

Comment:
blocking: these should be added to `exports` a few lines above so this also fixes the containerize issue

Reply:
Done with commit bf26d5473ca620522b70c9e6257837c0a712e649 - thanks for highlighting.